Documentation/ceph-cluster-crd.md

```diff
@@ -315,6 +316,7 @@ The following are the settings for Storage Class Device Sets which can be config
    * For example, provisioning may require topology spread constraints across zones, but the OSD daemons may require constraints across hosts within the zones.
 * `portable`: If `true`, the OSDs will be allowed to move between nodes during failover. This requires a storage class that supports portability (e.g. `aws-ebs`, but not the local storage provisioner). If `false`, the OSDs will be assigned to a node permanently. Rook will configure Ceph's CRUSH map to support the portability.
 * `tuneDeviceClass`: If `true`, because the OSD can be on a slow device class, Rook will adapt to that by tuning the OSD process. This will make Ceph perform better under that slow device.
+* `tuneFastDeviceClass`: If `true`, because the OSD can be on a fast device class, Rook will adapt to that by tuning the OSD process. This will make Ceph perform better under that fast device.
```

Comment:
How about adding an example of where it should be used. Something like:
 "For example, Ceph incorrectly detects Azure disks from the storage class "<insert>" as hdds, so you can improve Ceph performance by setting this to `true`."

At the same time, an example would be helpful for the slow device class for AWS.

Reply:
It's not Ceph's fault, please do not say that 🙏 , it's the Azure hypervisor's block layer reporting the device incorrectly then the Kernel sees a rotational drive, nothing wrong the Kernel either.

But +1 for an example :D

Reply:
Yes, good to clarify that Ceph is not the problem. How about this? (and still need the azure storage class example name):
"For example, Ceph cannot detect Azure disks as hdds from the storage class "(name)", so you can improve Ceph performance by setting this to true."

Reply:
Added an example for both `tuneDeviceClass` for AWS disks and `tuneFastDeviceClass`Azure disks.

Reply:
Actually I don't think we need the full yaml example. How about just adding a sentence here? Do we have a BZ for this issue? I didn't see it. Hopefully the BZ logs show what storage class name azure is using.
"For example, Ceph cannot detect Azure disks as ssds from the storage class "(name)", so you can improve Ceph performance by setting this to true."

The example for AWS for the slow tuning setting would be similar:
"For example, Ceph cannot detect AWS volumes as hdds from the storage class "gp2", so you can improve Ceph performance by setting this to true."

Reply:
Yes, we do have Bz for this, and the storage class Azure using is `managed-premium`.